pkg/cli/describe_helm.go

```diff
@@ -0,0 +1,211 @@
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/loft-sh/log"
+	"github.com/loft-sh/vcluster/config"
+	"github.com/loft-sh/vcluster/pkg/cli/find"
+	"github.com/loft-sh/vcluster/pkg/cli/flags"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type DescribeOutput struct {
+	Distro       string   `json:"Distro"`
+	Version      string   `json:"Version"`
+	BackingStore string   `json:"Backing Store"`
+	ImageTags    ImageTag `json:"Image tags"`
```

Comment:
I read over the issue discussion in Linear and didn't see anything about this, but the keys with spaces is not something I see often, e.g. "Backing Store" instead of "backingStore".  Not a big deal if this is how we want the output to look, just calling attention to it, in case we'd like to go with more common camel case.

Reply:
Yeah true, we should do camelCase here